libcontainer/integration/init_test.go

```diff
@@ -21,10 +22,14 @@ func init() {
 	runtime.GOMAXPROCS(1)
 	runtime.LockOSThread()
 	if err := libcontainer.StartInitialization(); err != nil {
-		// as the error is sent back to the parent there is no need to log
-		// or write it to stderr because the parent process will handle this
-		os.Exit(1)
```

Comment:
Looks like this is the code that was added in the first commit; perhaps it would make sense to move this commit (and the previous one) to the start to the list of commits.

Reply:
So, the first commit ("libct/int: remove logger from init") is a preparation for "init.go: move logger setup to StartInitialization". Logically, they belong together, but since test changes in libct/int are somewhat larger (they also involve a test case), I separate them out for easier review.

Now, the change in the first commit (""libct/int: remove logger from init") brings libcontainer/integration `func init()` in line with the main runc's `func init`. Later, we change `StartInitialization` to return error in some cases, so we remove the `os.Exit(1)`.

All I can do is to rearrange the commits so these two are following each other:
- libct/int: remove logger from init
- init.go: move logger setup to StartInitialization

Reply:
Now, the code added is correct (when it is added), but since "libct/StartInitialization: amend docs, do not error" it needs to be fixed. Thus the two changes. Hope that makes sense.